dev/devicelab/lib/tasks/plugin_tests.dart

```diff
@@ -240,8 +240,8 @@ class $dartPluginClass {
           if (target == 'ios') {
             // Plugins with versions lower than the app version should not have IPHONEOS_DEPLOYMENT_TARGET set.
             // The plugintest plugin target should not have IPHONEOS_DEPLOYMENT_TARGET set since it has been lowered
-            // in _reduceDarwinPluginMinimumVersion to 7, which is below the target version of 9.
-            if (podsProjectContent.contains('IPHONEOS_DEPLOYMENT_TARGET = 7')) {
+            // in _reduceDarwinPluginMinimumVersion to 10, which is below the target version of 11.
+            if (podsProjectContent.contains('IPHONEOS_DEPLOYMENT_TARGET = 10')) {
               throw TaskResult.failure('Plugin build setting IPHONEOS_DEPLOYMENT_TARGET not removed');
```

Comment:
Confirmed that with the test change and without `podhelper.rb` on master, the test fails (proving this test is validating the correct thing).
```
[plugin_test_ios] Process terminated with exit code 0.
Task result:
{
  "success": false,
  "reason": "Plugin build setting IPHONEOS_DEPLOYMENT_TARGET not removed"
}
```